### PR TITLE
Use a more recent version of mamba 0.25.0

### PR DESCRIPTION
**Pull request by Hind-M**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Hind-M** [comment]
@conda-forge-admin, please rerender

**Comment by Hind-M** [comment]
@conda-forge-admin, please rerender

**Comment by Hind-M** [comment]
@conda-forge-admin, please rerender

**Comment by Hind-M** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ requirements:
     - {{ posix }}coreutils         # [win]
     - {{ posix }}zip               # [win]
   host:
-    - libmamba >=0.13.1
+    - libmamba >=0.25.0
```
Comment (SylvainCorlay) [inline recipe/meta.yaml]:
```suggestion
    - libmamba >=0.25.0,<0.26
```

**Pull request merged**